runtime/local/service.go

```diff
@@ -50,25 +49,6 @@ func newService(s *runtime.Service, c runtime.CreateOptions) *service {
 
 	dir := s.Source
 
-	// For uploaded packages, we upload the whole repo
```

Comment:
Removed this logic as it's pretty much the exact duplicate as what happens above,